packages/web-pkg/src/components/FilesList/ResourceTiles.vue

```diff
@@ -42,7 +42,7 @@
         </oc-filter-chip>
       </div>
     </div>
-    <oc-list class="oc-tiles grid justify-start">
+    <oc-list class="oc-tiles grid justify-start gap-3">
```

Comment:
you got me confused: here you have `gap-3` for 1rem, in other places you had `gap-2` for 1rem. What's correct now?